recipes/conda-which/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "conda-which" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kelvinou01/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: a0acc563af988398fb2cf2ca1548d7903310035b2ab9a67c9849b7d02a1c788c
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+
+  run:
+    - python >=3.8
+    - conda
+    - termcolor
+
+test:
+  imports:
+    - conda_which
+  requires:
+    - pip
+  commands:
+    - pip check
+
+
+about:
+  home: https://github.com/kelvinou01/conda-which
+  license: MIT
+  license_file: LICENSE.txt
```

Comment:
This should be `license_file: LICENSE` to match what is in the tarball source distribution.